aospy/utils/times.py

```diff
@@ -155,18 +155,20 @@ def numpy_datetime_range_workaround(date):
     Parameters
     ----------
     date : datetime.datetime object
+    min_year : int
+        Minimum year described in the fileset
```

Comment:
It's not immediately obvious what "described" means.  And maybe "Earliest" instead of "Minimum".  Maybe an extra sentence explanation?